charts/dapr/crds/resiliency.yaml

```
@@ -64,6 +64,14 @@ spec:
                           type: integer
                         policy:
                           type: string
+                        retryOnCodes:
+                          type: array
+                          items:
+                            type: string
+                        ignoreOnCodes:
+                          type: array
+                          items:
+                            type: string
```

It is, at least to me, not immediately obvious what "ignore" means in this context. `noRetry` might be clearer to users. 

```suggestion
                        noRetryOnCodes:
                          type: array
                          items:
                            type: string
```

---

Done.